README.md

```diff
@@ -87,77 +89,56 @@ Alternatively, you can selectively import only the matchers you intend to use,
 and extend jest's `expect` yourself:
 
 ```javascript
-import {toBeInTheDOM, toHaveClass} from 'jest-dom'
+import {toBeInTheDocument, toHaveClass} from 'jest-dom'
 
-expect.extend({toBeInTheDOM, toHaveClass})
+expect.extend({toBeInTheDocument, toHaveClass})
 ```
 
 > Note: when using TypeScript, this way of importing matchers won't provide the
 > necessary type definitions. More on this [here](https://github.com/gnapse/jest-dom/pull/11#issuecomment-387817459).
 
 ## Custom matchers
 
-### `toBeInTheDOM`
+### `toBeEmpty`
 
 ```typescript
-toBeInTheDOM(container?: HTMLElement | SVGElement)
-```
-
-This allows you to assert whether an element present in the DOM container or not. If no DOM container is specified it will use the default DOM context.
-
-#### Using the default DOM container
-
-```javascript
-// add the custom expect matchers once
-import 'jest-dom/extend-expect'
-
-// ...
-// <span data-testid="count-value">2</span>
-expect(queryByTestId(container, 'count-value')).toBeInTheDOM()
-expect(queryByTestId(container, 'count-value1')).not.toBeInTheDOM()
-// ...
+toBeEmpty()
 ```
 
-#### Using a specified DOM container
+This allows you to assert whether an element has content or not.
 
 ```javascript
 // add the custom expect matchers once
 import 'jest-dom/extend-expect'
 
 // ...
-// <span data-testid="ancestor"><span data-testid="descendant"></span></span>
-expect(queryByTestId(container, 'descendant')).toBeInTheDOM(
-  queryByTestId(container, 'ancestor'),
-)
-expect(queryByTestId(container, 'ancestor')).not.toBeInTheDOM(
-  queryByTestId(container, 'descendant'),
-)
+// <span data-testid="not-empty"><span data-testid="empty"></span></span>
+expect(queryByTestId(container, 'empty')).toBeEmpty()
+expect(queryByTestId(container, 'not-empty')).not.toBeEmpty()
 // ...
 ```
 
-> Note: when using `toBeInTheDOM`, make sure you use a query function
-> (like `queryByTestId`) rather than a get function (like `getByTestId`).
-> Otherwise the `get*` function could throw an error before your assertion.
-
-### `toBeEmpty`
+### `toBeInTheDocument`
 
 ```typescript
-toBeEmpty()
+toBeInTheDocument()
 ```
 
-This allows you to assert whether an element has content or not.
+This allows you to assert whether an element is present in the document or not.
 
 ```javascript
 // add the custom expect matchers once
 import 'jest-dom/extend-expect'
 
 // ...
-// <span data-testid="not-empty"><span data-testid="empty"></span></span>
-expect(queryByTestId(container, 'empty')).toBeEmpty()
-expect(queryByTestId(container, 'not-empty')).not.toBeEmpty()
+// <span data-testid="count-value">2</span>
+expect(queryByTestId(container, 'count-value').toBeInTheDocument()
+expect(queryByTestId(container, 'count-value1')).not.toBeInTheDocument()
```

Comment:
I think that `expect(nullOrUndefined).toBeInTheDocument()` would fail before even having the chance to actually check for presence in the document. So this second example is not accurate because `queryByTestId(container, 'count-value1')` will return `null` or `undefined`, and neither of those is a html element.

I think that `expect(something).not.toBeInTheDocument()` only passes if `something` is an html or sag element, but it's not attached to the document. It won't work at all for things that are not html or svg elements.

So in this sense, it's not a direct replacement for `toBeInTheDOM`, and I think that's a good thing. The migration path is not simply "replace all uses of `toBeInTheDOM` with `toBeInTheDocument`.

Not sure though on how to approach this in the README example to reflect on both cases. And also I think we need a test that asserts that `toBeInTheDocument` indeed fails when given `null` or `undefined`.

Reply:
Sounds Good! I will work to take care of these!

Thank you @gnapse!